targets/TARGET_NUVOTON/TARGET_M480/objects.h

```diff
@@ -118,10 +118,6 @@ struct pwmout_s {
     uint32_t pulsewidth_us;
 };
 
-struct trng_s {
```

Comment:
It might be worth updating the Copyright (c) year at the beginning of these files (TARGET_M480/objects.h  and TARGET_NUC472/objects.h).

Reply:
Changed year to 2019.